src/north/cli/gscli/sonic.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import os
+
+from .base import Object, InvalidInput, Completer
+from pyang import repository, context
+import json
+import yang as ly
+import sysrepo as sr
+import base64
+import struct
+
+from prompt_toolkit.document import Document
+from prompt_toolkit.completion import WordCompleter, Completion, FuzzyCompleter
+
+TIMEOUT_MS = 10000
+
+class Sonic(Object):
+    XPATH = '/sonic-port:sonic-port/PORT/PORT_LIST'
+
+    def __init__(self, session, parent):
+        self.session = session
+        super(Sonic, self).__init__(parent)
+
+        @self.command()
+        def show(args):
+            if len(args) != 0:
+                raise InvalidInput('usage: show[cr]')
+            tree = self.session.get_subtree(self.XPATH)
```

Comment:
@cshivashgit 

You can use the following to make this work

```
tree = self.session.get_data(self.XPATH, 0, TIMEOUT_MS)
```

Not sure why get_subtree doesn't work.

By using the `-v`  option, I can see the following error message.

```
[ERR]: More subtrees match "/sonic-port:sonic-port/PORT/PORT_LIST".
```

It's not immediate but can you look into the root cause when you have time?

Reply:
@ishidawataru 
>tree = self.session.get_data(self.XPATH, 0, TIMEOUT_MS)
This worked .

We will look into the root cause and fix the issue.